pages/account/profile.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <!-- Test page, show logged in user -->
```

Comment:
This is a test page so will be removed, right, as the profile page is in KeyCloak?

Reply:
Yes, this is test